app/backend/pyproject.toml

```diff
@@ -15,6 +15,7 @@ dependencies = [
     "Jinja2==3.1.6",
     "luhn==0.2.0",
     "markdown-it-py==4.0.0",
+    "mypy>=1.18.2",
```

Comment:
Can we put it in dev reqs, I think that's a `uv` feature?

Reply:
done